.github/workflows/docker.yml

```diff
@@ -14,11 +14,14 @@ jobs:
       run: git rev-parse HEAD > packages/cosmic-swingset/lib/git-revision.txt
     - name: Save SDK_VERSION
       run: echo "SDK_VERSION=$(jq -r .version package.json)" >> $GITHUB_ENV
+    - name: Save MODDABLE_COMMIT_HASH
+      run: set $(git submodule status); echo "MODDABLE_COMMIT_HASH=$1" >> $GITHUB_ENV
```

Comment:
What on earth does this do? Or maybe don't tell me; I don't really want to use any more head-space on bash arcana.

```bash
set $(git submodule status);
```

Reply:
> What on earth does this do?

`set $words` assigns each whitespace-separated word to `$1`, `$2`, etc.

Pretend you never saw that.